src/ll.Translator.js

```diff
@@ -40,9 +44,13 @@ ll.Translator.static.bundle = function ( groups ) {
  * @return {Array[]} Array of Arrays of strings
  */
 ll.Translator.static.unbundle = function ( bundled ) {
-	return bundled.split( /\s*:!!!:\s*/ ).map( function ( groupString ) {
-		return groupString.split( /\s*:!!:\s*/ );
-	} );
+	var innerPattern = new RegExp( '\\s*' + this.innerSeparator + '\\s*' ),
```

Comment:
I had "TODO: Regex-escape the separator strings"

Reply:
I'll add that.